com.unity.formats.fbx/Editor/FbxExporter.cs

```diff
@@ -2877,6 +2877,12 @@ private FbxNode CreateFbxNode(GameObject unityGo, FbxScene fbxScene)
                         ExportLight (go, fbxScene, node);
                     } else if (compType == typeof(Camera)) {
                         ExportCamera (go, fbxScene, node);
+                    } else if (compType == typeof(SkinnedMeshRenderer))
```

Comment:
Either follow the convention in this function of
```
} else if (...) {
```
or follow the Unity convention of 
```
}
else if (...)
{
```
but not halfway!